homework/java/fyodorov/lecture2/task1/src/main/java/by/jrr/mytest/service/DogService.java

```diff
@@ -0,0 +1,22 @@
+package by.jrr.mytest.service;
+import by.jrr.mytest.bean.Dog;
+
+public class DogService {
+
+    Dog dog;
+
+    public DogService(Dog dog) {
+        this.dog = dog;
+    }
+
+    public String bark (){
+        return dog.getName () + " BARK!";
```

Comment:
You can create class DogLanguage with static fields like this:
```
public class DogLanguage() {
    public static final String BARKING = "BARK!";
}
```

Than in code you can address like this:
```
public String bark() {
        return String.format("%s %s", dog.getName(), DogLanguage.BARKING);
}
```